pydoctor/epydoc/markup/__init__.py

```diff
@@ -184,9 +184,22 @@ class DocstringLinker:
     target URL for crossreference links.
     """
 
+    def resolve_identifier(self, identifier: str) -> Optional[str]:
+        """
+        Resolve a Python identifier.
+        This will resolve the identifier like Python itself would.
+
+        @param identifier: The name of the Python identifier that
+            should be linked to.
+        @return: The URL of the target, or L{None} if not found.
+        """
+        raise NotImplementedError()
```

Comment:
Since we don't have a test for this...maybe add 
```suggestion
        raise NotImplementedError()  # pragma: no cover
```

Reply:
I'd rather exclude all `raise NotImplementedError` instances from coverage in the configuration.